tests/nlp/checks/train_test_validation/property_drift_test.py

```diff
@@ -181,7 +181,7 @@ def test_with_drift(self, dummy_multilabel_textdata_train_test):
         condition_results = check.conditions_decision(result)
         assert_that(condition_results, has_items(
             equal_condition_result(is_pass=False,
-                                   details="Failed for 1 out of 6 columns.\nFound 1 "
+                                   details="Failed for 1 out of 7 columns.\nFound 1 "
```

Comment:
why did that change?

Reply:
There was a funny bug - 
Some time ago we removed the language property from calculations because it wasn't good enough. And then we changed it to use fasttext instead of langdetect and reinserted it. However, the tests were not updated, and still didn't fail. Why? Because CI/CD didn't install fasttext, so it skipped it. So the tests failed locally, but not remotely.
So I re-added fasttext to dev-requirements.txt and that changed the outcomes of some tests.

Reply:
what why was the result skipping rather than failing... you understand that?

Reply:
Because that's how properties are built :) We skip properties that the user doesn't have the package for

Reply:
(We print a warning, not raising an error)

Reply:
oh
right
and not test per property